packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
```

Comment:
Ah, I took it out since I figured people would just use the `fontSize` CSS attribute at this point.